packages/babel-helper-compilation-targets/src/index.js

```diff
@@ -1,19 +1,19 @@
 // @flow
 
 import browserslist from "browserslist";
-import findSuggestion from "levenary";
-import invariant from "invariant";
+import { findSuggestion } from "@babel/helper-validator-option";
 import browserModulesData from "@babel/compat-data/native-modules";
 
 import {
   semverify,
   semverMin,
   isUnreleasedVersion,
   getLowestUnreleased,
+  v,
```

Comment:
Is there a reason we're naming this `v`? I'm generally in favor of descriptive names, and I worry that its meaning will be obscured for future maintainers.

Reply:
+1 If `validator` is too verbose, we can still do `import { validator as v }`: at least you don't have to check into another file what `v` means.